### DISCUSSION TITLE
RaspberryMatic als ova installiert 3.69.6.20230407.ova

### DISCUSSION THREAD

**Discussion by Novell2** [OpenCCU/OpenCCU#2313] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen, 
wie kann auf die Version 3.69.7x erneuert werden in der Systemsteuerung und mit welchem Downloadfile? Installiert hatte ich das vor ca 2Montaen als OVA in esxi 8.
Möchte wenn's geht nicht eine neue OVA (neue VM) installieren, sondern die "alte" updaten.
Besten Dank!
Gruss
Novell2

**Comment by Novell2** [comment]
Hallo zusammen,
selber gelöst durch das update file RaspberryMatic 3.69.7.xxxxx-ova.zip

Danke und Tsachüss!

**Comment by MichaelN0815** [comment, marked as answer]
Beim nächsten Mal einfach auf die feinen Details achten:

![grafik](https://github.com/jens-maus/RaspberryMatic/assets/68965895/44fe55d3-daea-4e91-a83b-bfbc4c5aca37)

![grafik](https://github.com/jens-maus/RaspberryMatic/assets/68965895/8b6ff37c-d541-46f2-be73-80d665bd6a54)

**Comment by Novell2** [comment]
Ja, danke trotzdem, wünsche Ihnen eine gute Woche!




Von meinem/meiner Galaxy gesendet



-------- Ursprüngliche Nachricht --------
Von: MichaelN0815 ***@***.***>
Datum: 09.05.23 19:36 (GMT+01:00)
An: jens-maus/RaspberryMatic ***@***.***>
Cc: Novell2 ***@***.***>, Author ***@***.***>
Betreff: Re: [jens-maus/RaspberryMatic] RaspberryMatic als ova installiert 3.69.6.20230407.ova (Discussion #2313)


Beim nächsten Mal einfach auf die feinen Details achten:

[grafik]<https://user-images.githubusercontent.com/68965895/237175311-44fe55d3-daea-4e91-a83b-bfbc4c5aca37.png>
